tests/test_ingest.py

```diff
@@ -44,6 +45,7 @@ class HscIngestTestCase(IngestTestBase, lsst.utils.tests.TestCase):
                                       "transmission_sensor", "transmission_filter", "transmission_atmosphere")
     ingestDir = os.path.dirname(__file__)
     instrumentClassName = "lsst.obs.subaru.HyperSuprimeCam"
+    filterLabel = lsst.afw.image.FilterLabel(physical="HSC-I", band="i")
```

Comment:
This appears to be unused.

Reply:
Never mind, forgot this is defined by `IngestTestBase`.

Reply:
The parent class `IngestTestBase` uses it.